client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -193,55 +200,50 @@ var CheckoutThankYou = React.createClass( {
 
 		return (
 			<div>
-				{ React.createElement( componentClass, {
-					selectedSite: selectedSite,
-					isFreeTrial: this.freeTrialWasPurchased(),
-					locale: i18n.getLocaleSlug(),
-					domain: domain
-				} ) }
+				<CompactCard className="checkout-thank-you__features-header">
+					{ this.translate( "Get started with your site's new features" ) }
+				</CompactCard>
+
+				<div className="checkout-thank-you__purchase-details-list">
+					{ React.createElement( componentClass, {
```

Comment:
Not introduced by you, but we can get rid of the `React.createElement` call here and just use JSX:

```js
<ComponentClass
  selectedSite={ selectedSite }
  isFreeTrial={ this.freeTrialWasPurchased() }
  domain={ domain } />
```